openreview/tools.py

```
@@ -61,9 +71,9 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
             return profile
 
         else:
-            raise openreview.OpenReviewException('There is already a profile with this first: {0}, middle: {1}, last name: {2}'.format(first, middle, last))
+            raise openreview.OpenReviewException('There is already a profile with this first and last name: \"{first} {last}\"'.format(first=first, last=last))
     else:
-        return profile
+        raise openreview.OpenReviewException('There is already a profile with this email address: {}'.format(email))
```

You are right, we should raise the exception.